tests/vendor/test_hook.py

```diff
@@ -1,8 +1,8 @@
 import mock
 
 from ddtrace.compat import reload_module
-from ddtrace.utils.hook import deregister_post_import_hook
-from ddtrace.utils.hook import register_post_import_hook
+from ddtrace.vendor.hook import deregister_post_import_hook
```

Comment:
wait, is this actually used anywhere? if there are only tests for it, delete the entire file?

Reply:
There's also `internal/import_hooks.py` that does a similar thing and it's just tested but not used. Is it worth investigating why these two modules are there in the first place?